packages/analitics/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@frontity/analytics",
+  "version": "0.1.0",
+  "description": "Analytics utilities for »Frontity",
```

Comment:
I'd say more something like: `This package contains the base types and actions to build analytics packages for Frontity`

Reply:
Much better!